### PR TITLE
chore: fix security issue unencrypted volume

### PR DESCRIPTION
**Pull request by xshot9011**
# Submit a pull request :rocket:

Thank you for help us contribute! Please give us more information about this PR.

---

### Changed

- Update module `efs` from version `v1.0.1` to `v1.0.3`
- Enable encryption in module `efs` with `encrypted = true`

## Why :pleading_face:

- Encryption in EFS

### CODE REVIEW

File: examples/complete/vpc.tf [hunk 1]
```diff
@@ -1,16 +1,22 @@
...
-  public_subnets               = ["10.105.0.0/24", "10.105.1.0/24", "10.105.2.0/24"]
-  database_subnets             = ["10.105.20.0/23", "10.105.22.0/23", "10.105.24.0/23"]
-  availability_zone            = ["ap-southeast-1a", "ap-southeast-1b", "ap-southeast-1c"]
-  is_enable_dns_hostnames      = true
-  is_enable_dns_support        = true
-  is_create_nat_gateway        = true
-  is_enable_single_nat_gateway = true
-  account_mode                 = "hub"
-  tags                         = var.tags
+  source = "git@github.com:oozou/terraform-aws-vpc.git?ref=v1.1.7"
```
Comment (waruwat-dev) [inline examples/complete/vpc.tf:2]: เปลี่ยนเป็น public

File: efs.tf [hunk 2]
```diff
@@ -1,10 +1,11 @@
 # EFS
 module "efs" {
-  source = "git::ssh://git@github.com/oozou/terraform-aws-efs.git?ref=v1.0.1"
+  source = "git::ssh://git@github.com/oozou/terraform-aws-efs.git?ref=v1.0.3"
```
Comment (waruwat-dev) [inline efs.tf:3]: เปลี่ยนไปใช้ public

**Review by waruwat-dev — CHANGES_REQUESTED** [review]

File: examples/complete/vpc.tf (hunk 1 above)
Comment (xshot9011) [reply examples/complete/vpc.tf:2]: Sure

File: efs.tf (hunk 2 above)
Comment (xshot9011) [reply efs.tf:3]: Sure

**Review by waruwat-dev — APPROVED** [review]

**Pull request merged**